src/PlaneWaveBasis.jl

```diff
@@ -138,6 +128,18 @@ function PlaneWaveBasis(model::Model{T}, Ecut::Number,
         kweights, ksymops, fft_size, grids, opFFT, ipFFT, opIFFT, ipIFFT
     )
 end
+function PlaneWaveBasis(model::Model{T}, Ecut::Number, kgrid_size::AbstractVector;
+                        bzmesh_symmetry=:full, kwargs...) where {T <: Real}
```

Comment:
Don't need to {T}

Reply:
I think just make `bzmesh_symmetry` a bool. People needing finer control just call the other constructor.